<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgPrefWebServerDlg</class>
 <widget class="QWidget" name="DlgPrefWebServerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Web Server Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxEnableWebServer">
     <property name="text">
      <string>Enable web server</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxNetwork">
     <property name="title">
      <string>Network</string>
     </property>
     <layout class="QFormLayout" name="formLayoutNetwork">
      <property name="labelAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labelBindAddress">
        <property name="text">
         <string>Bind address:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxBindAddress"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxPort">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxAuthentication">
     <property name="title">
      <string>Authentication</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutAuth">
      <item>
       <widget class="QCheckBox" name="checkBoxEnableAuthentication">
        <property name="text">
         <string>Require authentication</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutAuthentication">
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelUsername">
          <property name="text">
           <string>Username:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditUsername"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPassword">
          <property name="text">
           <string>Password:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayoutAuthWarning">
        <item>
         <widget class="QLabel" name="labelAuthWarningIcon">
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelAuthWarning">
          <property name="text">
           <string>Authentication without HTTPS sends credentials unencrypted.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxTls">
     <property name="title">
      <string>Transport Layer Security</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayoutTls">
      <item>
       <widget class="QCheckBox" name="checkBoxUseHttps">
        <property name="text">
         <string>Use HTTPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxAutoGenerateCertificate">
        <property name="text">
         <string>Auto-generate certificate and key</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayoutTls">
        <property name="labelAlignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelCertPath">
          <property name="text">
           <string>Certificate:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayoutCert">
          <item>
           <widget class="QLineEdit" name="lineEditCertPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrowseCert">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelKeyPath">
          <property name="text">
           <string>Private key:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayoutKey">
          <item>
           <widget class="QLineEdit" name="lineEditKeyPath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrowseKey">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelCaBundle">
          <property name="text">
           <string>CA bundle:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayoutCaBundle">
          <item>
           <widget class="QLineEdit" name="lineEditCaBundlePath"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonBrowseCaBundle">
            <property name="text">
             <string>Browse...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
